chips/earlgrey/src/registers/README.md

```diff
@@ -0,0 +1,16 @@
+# Earlgrey Register Definitions
+
+The files in this directory are auto-generated register definitions for
+Earlgrey peripherals.  These files were auto-generated from the OpenTitan
+codebase as follows:
+
+```bash
+$ cd $OPENTITAN_TREE
+$ git checkout Earlgrey-M2.5.2-RC0
```

Comment:
We want to make sure this matches `OPENTITAN_SUPPORTED_SHA` in the Makefile.

Let's get the non README changes (SHA bump and linker script) from https://github.com/tock/tock/pull/3586 merged, then this PR can go next

Reply:
That is the plan - this was always meant to go after #3586.